pages/api/_common.ts

```diff
@@ -1,4 +1,4 @@
-import { ethers, Contract, BigNumber } from "ethers";
+import { ethers, Contract } from "ethers";
```

Comment:
this is unrelated but i could not build, due to linting error. dont know how this passed ci in previous pr

Reply:
This now fixed 👍